README.md

```diff
@@ -328,23 +326,32 @@ In order to build llama.cpp you have four different options.
         make
         ```
 
+  - Notes:
+    - For faster compilation, add the `-j` argument to run multiple jobs in parallel. For example, `make -j 8` will run 8 jobs in parallel.
+    - For faster repeated compilation, install [ccache](https://ccache.dev/).
+    - For debug builds, run `make LLAMA_DEBUG=1`
+
 - Using `CMake`:
 
-    ```bash
-    cmake -B build
-    cmake --build build --config Release
-    ```
+  ```bash
+  cmake -B build
+  cmake --build build --config Release
+  ```
+
+  **Notes**:
 
-    **Note**: for `Debug` builds, there are two cases:
+    - For faster compilation, pass arguments to the internally used build tool via `--`. For example when internally using `make`, `cmake --build build --config Release -- -j 8` will pass `-j 8` to `make` and result in 8 jobs being run in parallel.
```

Comment:
`cmake` supports `-j` without needing to use `--`. I would expect that to be better since it will work with any generator.

Reply:
Thank you, I didn't see the option in the `--help` but it does indeed seem to exist. (Added with v3.12, llama.cpp minimum is v3.14.)